Proposals/TypeNetwork_ParametricAxes/ProposalSummary_ytuc.md

```diff
@@ -39,7 +39,8 @@ setting to adjust to space, resolution or orientation.
 **UI recommendations:** Users may choose to program a variant in connection to direct or
 conjunctive input for a page description language, or via a user interface.
 
-**Script or language considerations:** Can be used for all scripts.
+**Script or language considerations:** This axis is primarily intended for Latin script and other scripts that may have a case, or character group intended to remain aligned relative to the Latin Uppercase throughout the variable space. Typically, this would apply to scripts in which some letters generally have a core height the same or close to the Latin Uppercase in the font.
```

Comment:
I find the wording "may have a case" odd: it sounds like "may have a rational for...". I suggest you change to something like one of the following:

- "may have case"
- "may have bicameral case"
- "may have a case distinction"